bc_obps/reporting/service/report_supplementary_version_service.py

```diff
@@ -456,6 +457,17 @@ def clone_report_version_facility_product_data(
         old_facility_report_to_clone: FacilityReport,
         new_facility_report: FacilityReport,
     ) -> None:
+        # Clone the parent ReportEmissionAllocation
+        emission_allocation_to_copy = ReportEmissionAllocation.objects.filter(
+            report_version_id=old_report_version,
+            facility_report=old_facility_report_to_clone,
+        ).first()
+        if emission_allocation_to_copy:
+            emission_allocation_to_copy.pk = None
+            emission_allocation_to_copy._state.adding = True
+            emission_allocation_to_copy.report_version = new_report_version
```

Comment:
Doesn't this need a fkey set to the new facility_report object?

Reply:
Ah it definitely should